Zoo/src/JosPlateau/Animals.java

```diff
@@ -0,0 +1,27 @@
+package JosPlateau;
+
+/**
+ * Created by admin-batch-a on 7/15/17.
+ */
+ public class Animals {
+public void localZoo (){
+
+    String[] animalNames = new String[10];
+
+    animalNames [0] = "Stanstead";
+    animalNames [1] = "Gatwick";
+    animalNames [2] = "Heathrow";
+    animalNames [3] = "Luton";
+    animalNames [4] = "Birmingham";
+    animalNames [5] = "Newscastle";
+    animalNames [6] = "Sutherland";
+    animalNames [7] = "Manchester";
+    animalNames [8] = "LeedsBradford";
+    animalNames [8] = "RobinHood";
+
+
+            for (int i = 0; i < animalNames.length; i++) {
```

Comment:
Hi Tim, I am not sure what this code is doing. This code would assign "String Number 1", etc. Kindly review it and reassign it.